README.md

```diff
@@ -82,6 +82,10 @@ export async function prerender(data) {
 
 A context provider that provides the current location to its children. This is required for the router to function.
 
+Props:
+
+-   `limit?: string | RegExp` - Sets a limit on the paths that the router will handle (intercept). If a path does not match the limit, either by starting with the provided string or matching the RegExp, the router will ignore it and default browser navigation will apply.
```

Comment:
The name limit bothers me a bit, I would prefer something like include/exclude personally

Reply:
Hm, "include" means something quite different though. To include something is to have it be part of a whole; this _limits_ the whole to only the matching value.

There might be a better name (haven't yet found/thought of one myself), though I think "include" would give a very different idea than what we're going for.

Reply:
Maybe `scope`? I'm not dead-set against `include`, can do it, though not my fav.